src/module/actor/character/document.ts

```diff
@@ -315,10 +320,11 @@ class CharacterPF2e<TParent extends TokenDocumentPF2e | null = TokenDocumentPF2e
             },
         };
 
-        // Base ability scores
-        for (const abbrev of ABILITY_ABBREVIATIONS) {
-            systemData.abilities[abbrev] = mergeObject({ value: 10 }, systemData.abilities[abbrev] ?? {});
-        }
+        // Base saves structure
+        systemData.saves = mergeObject(
```

Comment:
Seems saves are set up lower down in 376. Would it not make sense to move some of that up here so that we're not double reassigning base save objects?